OmiseSwift/Transfer.swift

```diff
@@ -20,6 +20,7 @@ public struct Transfer: OmiseResourceObject {
     public var createdDate: Date
     
     public var status: TransferStatus
+    public let isFailFast: Bool
```

Comment:
It's the property on `Transfer` so it sounds like `isFailFast` is OK to me. But we can use the name `shouldFailFast` on the `TransferParams` instead?